### DISCUSSION TITLE
The custom domain is already taken. Except it's not

### DISCUSSION THREAD

**Discussion by andreineculau** [community/community#23569] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,<br>
I’m faced with a weird situation where Github Pages complains that my domain ysoftware.se is already taken.</p>
<p>The story is that weeks ago I have created a repository, enabled Github Pages with ysoftware.se as a custom domain, worked fine, but realized I should have actually rename a github org instead of creating a new one and moving repos. So I have deleted the github pages repo (uncertain if i removed the custom domain first) and the organization.</p>
<p>Now, whenever I want to enable the custom domain, I get: The custom domain <code>[ysoftware.se](http://ysoftware.se/)</code> is already taken. Check out <a href="https://docs.github.com/articles/troubleshooting-custom-domains#cname-errors" class="inline-onebox" rel="noopener nofollow ugc">Troubleshooting custom domains and GitHub Pages - GitHub Docs</a> for more information.</p>
<p>I have verified that ysoftwareab organization owns ysoftware.se and I’ve also verified the domain for github pages. I have even tried to push manually a CNAME file but I get the same error via mail.</p>
<p>Since I own ysoftware.se I can actually set up the A records etc so that github pages serves ysoftware.se, but when I request <a href="https://ysoftware.se" rel="noopener nofollow ugc">https://ysoftware.se</a> I only get a 404 page from github . So it’s really pointing into the void, and not to some repository’s actual github pages’ content.</p>
<p>I contacted support last week, got a first reply that my DNS is not set up correctly, but that’s obviously not what the issue at hand is. It has been silent since then. I upgraded my account to Team, not sure if that escalates somehow my ticket, but :crossed_fingers:.</p>
<p>While waiting for an update from github support, I was wondering if others faced similar issues, and after searching the support community, I see that a handful of people were as unlucky as I am, but it looks like many/all cases have been manually handled by github staff. So here I am pleading my case :pray:</p>
<p>Thanks in advance,<br>
Andrei</p>

**Comment by yoannchaudet** [comment]
<p>Hello,</p>
<p>I just saw your support ticket and we may have an idea as to why the domain was not released how it should have.</p>
<p>An engineering on the team released the domain manually. Let us know if you still face issues.</p>

**Reply by Pompey21** [reply]
Hi @yoannchaudet I have a similar issue - i have verified my domain yet I still cannot use it (my issue is #1731078)

**Reply by shpanwar12** [reply]
I have similar issue can't add custom domain name and all DNS records already updated and domain name is verified.

**Reply by yoannchaudet** [reply]
@shpanwar12 Once it is verified a domain is not cleared right away today (we will improve that in the future and potentially provide an option). It takes 7 days for a domain to be completely released after you verified it. If you need help faster, please open a [support request](https://support.github.com/features/pages).

**Reply by siminliu99** [reply]
@yoannchaudet I have the same issue. Have been waiting for the domain to clear for more than 2 weeks... (ticket #1919851)

**Reply by leowebguy** [reply]
I'm having the same issue

**Reply by h4m5t** [reply]
I'm having the same issue

**Reply by swapnilsingh018** [reply]
> @yoannchaudet I have the same issue. Have been waiting for the domain to clear for more than 2 weeks... (ticket #1919851)

Hey, Your website is working now, How did you do it?

**Reply by h4m5t** [reply]
Hello,everyone,after a few days, it was solved.

**Reply by FANXING-0710** [reply]
I'm having the same issue

**Reply by awangga** [reply]
@yoannchaudet  i have same issue. my subdomain hijack by another people https://github.com/Wowyyyy/iteung.ulbi.ac.id
its a same case in this article : https://medium.com/@jehy/hijacking-domain-using-github-pages-41c80ac57523

i have verified my domain in : https://github.com/iteungweb/ but cannot use or add the domain in the https://github.com/iteungweb/iteungweb.github.io repositori
![image](https://github.com/community/community/assets/11188109/cbb2a57a-e9e7-4494-9447-5718a7dfbf32)

**Reply by kirakoki** [reply]
I have the same issue here! Can someone from the engineering team help me solve this ? @yoannchaudet

**Reply by 5v3n** [reply]
My gh-pages website was defaced by an attacker too! I verified my domain in my account, the txt entries show up when checked. Still my website is down since a couple of days & github claims they can't see the txt records (although my account settings clearly say it's verified via that method). Why is it so easy to steal my gh-pages website for a random attacker, but so difficult to get it back?

**Reply by missflash** [reply]
I have a same issue. My custom domain (missflash.com) was successfully verified, but unknown site was connected. 
[yoannchaudet](https://github.com/yoannchaudet)

**Reply by FANXING-0710** [reply]
Thank you for your answer, I've solved it.

**Reply by firstmover** [reply]
Hi @yoannchaudet, it seems that my GitHub page is also under a taken-over attack. Could you help me look into this issue? My ticket number is #2837558. I have messaged GitHub support for around a week but I did not get any reply. Thank you for your assistance!

**Reply by yashrajbains** [reply]
Hi @yoannchaudet , sorry to be another user tacking on but I am facing the same exact issue! Messaged GitHub support although no response yet. Thank you!

**Reply by ns-mkusper** [reply]
Hey @yoannchaudet , same issue here!

**Reply by JusperLee** [reply]
@yoannchaudet , same issue here! cslikai.cn

**Comment by andreineculau** [comment]
<p>All green now. As they say in Swedish: tusen tack! = a thousand thanks!<br>
If there’s any detail to share, to keep in mind for next time something like this happens, please do! Maybe I should have done X, then Y, etc. Stuff like this.</p>

**Comment by rbeer** [comment]
<p>Having the same issue. Added the domain to my personal account, but needed it on an origanization. Removed it from the personal account and added it to the organization, without issues. Both via Settings → Pages.</p>
<p>It does not work when I’m now trying to add the domain directly to a Repository via its Settings → Pages. I have removed the domain from both accounts before, but I’m still getting the error.</p>

**Comment by yoannchaudet** [comment]
<p>Hello!</p>
<p>Please <a href="https://support.github.com/contact">reach out to the support</a> with the custom domain in question. They will be able to assist you.</p>

**Comment by yoannchaudet** [comment]
<p>You are very welcome.</p>
<p>In that specific case I believe you hit a bug and contacting support was a great call. Last I checked it was still being investigated (some event remained in a queue instead of being processed, hence the absence of release on the custom domain).</p>

**Comment by rbeer** [comment]
<p>I already did and they made me aware that I had the domain linked to another repo. I uploaded a CNAME file 3 years ago and wasn’t aware back then that doing so links the domain; so I forgot about it.</p>

**Comment by jrtib** [comment]
Just saw this thread and I'm currently having the same issue. I only have 1 repo that I've ever used this custom domain for but took the site down for maintenance. When I tried to re-publish my site with the custom domain used before only for the 1 repository for my website I have this error stating that the custom domain is already taken.

**Reply by kutyel** [reply]
I'm having the same issue as well, even after I deleted my whole kutyel.github.io repo (the one using previously the same CNAME file), the domain keeps getting rejected even when I followed the steps in https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages to verify my domain. Opened a github support ticket and hope they fix it soon 🤞🏻

**Reply by jrtib** [reply]
I actually discovered that my custom domain had been hijacked by another user before I had verified it with GitHub myself, so basically they were able to confirm it was my custom domain and then released it from the squatter repo to my repo. Once a custom domain is assigned to a repo apparently you get this error if you try to add it to another repo (or at least in my case). 

Has your website had any unusual activity recently? I would check for a domain squatter and if there is someone using your custom domain then support will probably be able to release it to you since you mentioned you've already verified it.

**Reply by kutyel** [reply]
I checked github but could not find any other repo using my domain in a CNAME file, so hope support will be able to fix it 😬

**Comment by xingarr** [comment]
Hey man
you can review this url:
https://stackoverflow.com/questions/69432084/how-to-decode-a-solana-transaction

**Comment by wndxlori** [comment]
I'll just add my gripe.  Same thing happened to one of my custom domains.  I have since verified the domain (personal and org), but it takes 7 days (?!?!?!?!) for the domain to be released?  That's some serious BS process.  Seconds to steal, a week to recover. Anyway, I submitted a ticket, to see if I can't get someone to fix it sooner.

**Reply by wndxlori** [reply]
Support responded, and I have my domain back again. What a PITA.

**Comment by firstmover** [comment]
A similar issue happened to my GitHub page and custom website as well. Sent a ticket to GitHub Support. And I hope they can help resolve this issue soon.

**Reply by Merrizee** [reply]
Is it solved? How long did it take

**Reply by firstmover** [reply]
I feel GitHub has helped me remove the malicious user who has taken over my domain (ie, if I go to my domain: `yingchengliu.com`, it is 404). But I still cannot add `yingchengliu.com` to my GitHub page. GitHub page still complains that it is taken. It's two month since I first report the problem.

**Reply by firstmover** [reply]
My problem was solved (3 months since I first reported the problem). GitHub now has a new process to verify your custom domain: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages . We first verify the domain in the user profile. Then go to your IO repo and add the domain.

**Comment by drewpotter** [comment]
I am having the same problem. Please help me.

**Comment by shcherbakaite** [comment]
Yeah, if you are not on paid plan they will tell you to screw off...

**Comment by ianderrington** [comment]
Where did you file support?
I have that issue now too

**Reply by shcherbakaite** [reply]
Under Profile context menu, GitHub Support, My Tickets (top right), New Ticket. First, you will have to go through CoIdiot AI bot and convince that it's stupid. Once it gives up on you, it will let you file a ticket. https://support.github.com/tickets/personal/0

**Comment by iamprincechaudhary** [comment]
A similar issue happened to my GitHub page and custom website as well. Sent a ticket to GitHub Support. And I hope they can help resolve this issue soon.

**Comment by 0xpsyduck** [comment]
I went through the same domain takeover when I made my repo private.
I have solved it for now by adding a verified domain under profile settings.

My question is how were the attackers able to verify my domain or anyone's domain as their own to Github?
Where is Github lacking?

**Comment by arxjei** [comment]
Solution that worked for me:

1. Turn off GitHub Pages in the old repository (where the domain was assigned)
2. Go to your Profile Picture → Settings
3. Under "Code, planning, and automation" → click Pages
4. Click "Add a domain" and enter your domain
5. Add the DNS TXT record they give you
6. Wait for verification (takes just a few seconds)
7. Go to your new repository and add the domain there

It worked! The domain verification at the profile level releases it from the old repo automatically.
